arch/arm/boot/dts/overlays/README

```diff
@@ -431,6 +431,10 @@ Info:   Configures the Allo Katana DAC audio card
 Load:   dtoverlay=allo-katana-dac-audio
 Params: <None>
 
+Name:   allo-boss2-dac-audio
```

Comment:
But actually this whole entry belongs before allo-digione - you got the Makefile order right,